quantum_mechanics.tex

```diff
@@ -0,0 +1,43 @@
+SymPy has extensive capabilities for symbolic quantum mechanics in the
+\verb|sympy.physics.quantum| subpackage. At the base level, this subpackage has
+Python objects to represent the different mathematical objects relevant in
+quantum theory \cite{sakurai}: states (bras and kets), operators (unitary,
+hermitian, etc.) and basis sets as well as operations on these objects such as
+tensor products, inner products, outer products, commutators, anticommutators,
+etc.  The base objects are designed in the most general way possible to enable
+any particular quantum system to be implemented by subclassing the base
+operators to provide system specific logic.
+
+The \verb|quantum| subpackage has a general purpose \verb|qapply| function that
+is capable of applying operators to states symbolically as well as simplifying
+a wide range of symbolic expressions involving different types of products and
+commutator/anticommutators. The state and operator objects also have a rich API
+for declaring their representation in a particular basis. This includes the
+ability to specify a basis for a multidimensional system using a complete set
+of commuting Hermitian operators.
+
+On top of this base set of objects, a number of specific quantum systems have
+been implemented. First, we have implemented the traditional algebra for
+quantum angular momentum \cite{zare}. This allows the different spin operators
+($S_x$, $S_y$, $S_z$) and their eigenstates to be represented in any basis and
+for any spin quantum number. Facilities for Clebsch-Gordan Coefficients, Wigner
```

Comment:
Maybe expand on this, replace this sentence with:
A rotation operator, which may be defined symbolically or numerically, is also implemented to rotate spin eigenstates. Functionality for coupling and uncoupilng of arbitrary spin states is provided, including symbolic representations of Clebsch-Gordon coefficients and Wigner symbols.